packages/babel-helper-compilation-targets/test/__snapshots__/targets-parser.spec.js.snap

```diff
@@ -37,6 +37,7 @@ Object {
   "ios": "10.3.0",
   "node": "13.2.0",
   "opera": "48.0.0",
+  "opera_mobile": "73.0.0",
```

Comment:
Although `op_mob` is set to 45, `op_mob 45` is still an unknown browser for `browserslist`, so it is resolved to `opera_mobile 73.0.0`. This probably won't affect end users unless they use `esmodules: "intersect"` with `opera_mobile` only queries.

Reply:
It's unfortunate, but it's probably the best we can do.

Reply:
> This probably won't affect end users unless they use esmodules:

This statement is a bit concerning to me, but I don't really understand the restriction.  Could you explain further?  Is there a tweak to `browserslist` that could alleviate it?

Reply:
If you are using `esmodules: true`, the `"opera_mobile": "73.0.0"` here is no op because Opera mobile 73 should cover every features implemented by other old browsers listed here such as Firefox 60 and Chrome 61.

If you want to fix this in `browerslist`, the upstream library https://github.com/browserslist/caniuse-lite should contain every opera mobile versions, so `browserslist` can actually resolve `op_mob 45` to opera mobile 45.

Reply:
Is my understanding that this is not a regression introduced by this PR correct? And the old babel behavior was to just map opera mobile to opera desktop, which also had wrong results since the mapping was wrong.

Reply:
It's a bug fix with no regression IMO:

- With current Babel and `browserslist` < 4.20.9, Babel and browserslist essentially had the same incorrect assumption of equating the two.  The net effect was sort of just ignoring Opera Mobile as an independent target.
- With current Babel and 4.20.9, Babel is doing some extra transpiling for any target that includes op_mob from browserslist
- With this PR and 4.20.9, the extra transpiling goes away

I cannot think of a query at the moment that would result in any sort of regression or breakage.